src/ifd-nfc.c

```diff
@@ -444,8 +519,12 @@ IFDHGetCapabilities(DWORD Lun, DWORD Tag, PDWORD Length, PUCHAR Value)
       *Value  = 1;
       *Length = 1;
       break;
+#if defined(HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD) && HAVE_DECL_TAG_IFD_POLLING_THREAD
```

Comment:
The problem is that in theory `TAG_IFD_STOP_POLLING_THREAD` could be declared by something else than `#define ...`. If so, this is done by a non-preprocessor directive (e.g. `const int ...`), which the preprocessor does not understand. However, the autoconf macro I used is exactly what recognizes all of that including a define, a function or a global variable. That's why I am using `HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD` instead of `TAG_IFD_STOP_POLLING_THREAD`